src/components/cartCard/cartCard.js

```diff
@@ -13,7 +12,61 @@ export default function CartCard({ cartitem }) {
             }
         });
         setUser({ ...user, cart: res.data.cart });
+    }
+    const moveToWishlist = async () => {
+        if (user.wishlist.length > 0) {
+            if (!user.wishlist.some(item => item._id === _id)) {
+                //if not in wishlist
+                const res = await axios.post("/api/user/wishlist", { product }, {
+                    headers: {
+                        authorization: encodedToken
+                    }
+                });
+                console.log(res.data, "...", product);
+                setUser({ ...user, wishlist: res.data.wishlist });
+                handleRemove();
+            }
+        } else {
+            const res = await axios.post("/api/user/wishlist", { product }, {
+                headers: {
```

Comment:
suggestion: you can keep all the Apiurl in a different file and use variables everywhere.

Reply:
noted this for future code practices, thanks.